.github/workflows/build_robotframework_aio.yml

```diff
@@ -124,7 +124,7 @@ jobs:
           curl -X POST https://api.github.com/repos/test-fullautomation/robotframework-documentation/dispatches \
           --header 'authorization: Bearer ${{ secrets.PAT_THONG }}' \
           -H 'Accept: application/vnd.github.everest-preview+json' \
-          --data '{"event_type": "Document", "client_payload": { "repository": "'"$GITHUB_REPOSITORY"'", "branch" : '"$doc_branch"'}}'
+          --data '{"event_type": "Document", "client_payload": { "repository": "'"$GITHUB_REPOSITORY"'", "branch" : "'"$doc_branch"'"}}'
```

Comment:
Hi Thong,

Please add the response verification (status code) to ensure that the request is sent properly as our discussion.

Thank you,

Reply:
Hi Ngoan,

Thank you for your comment. I have updated it.

Best regards,
Thong Hua